pkg/portForward/portForward.go

```diff
@@ -106,6 +103,23 @@ func (o *PFClient) StopPortForwarding() {
 	o.finishedChan = nil
 }
 
+func (o *PFClient) GetForwardedPorts() map[string][]int {
+	return o.appliedEndpoints
+}
+
+func (o *PFClient) GetPortsToForward(devFileObj parser.DevfileObj) (map[string][]int, error) {
+
+	// get the endpoint/port information for containers in devfile and setup port-forwarding
```

Comment:
Nit: the part `and setup port-forwarding` is inaccurate since we aren't setting up port forwarding in this method.